Makefile

```diff
@@ -364,7 +364,7 @@ deploy: kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/c
 	$(SED) -i -r 's~ebpf-agent:.+~ebpf-agent:main~' ./config/manager/manager.yaml
 	$(SED) -i -r 's~flowlogs-pipeline:.+~flowlogs-pipeline:main~' ./config/manager/manager.yaml
 	$(SED) -i -r 's~console-plugin:.+~console-plugin:main~' ./config/manager/manager.yaml
-	$(KUSTOMIZE) build config/openshift | sed -r "s/openshift-netobserv-operator\.svc/${NAMESPACE}.svc/" | kubectl apply -f -
+	$(KUSTOMIZE) build config/openshift | sed -r "s/openshift-netobserv-operator\.svc/${NAMESPACE}.svc/" | kubectl create -f -
```

Comment:
hey @jpinsonneau @msherif1234 @OlivierCazade : I'm hitting again the same issue that Julien had previously, with `kubectl apply` failing due to send too large annotations. (and `create` passes)
So temporarily I'm using create; but we should find a more durable fix for that (previously we did remove some of the CRD descriptions to make it pass, but it was just a matter of time to see the problem rise again)

Are we too verbose in the doc? Is the CRD too big? idk how to fix that. Or perhaps we should just work a bit more with the Makefile to do `create` / `delete` with the CRD (instead of `apply`) and keep `apply` for all other resources... Any idea?

Reply:
try with `kubectl apply --server-side` see if that helps I added more comments in
https://issues.redhat.com/browse/NETOBSERV-1613